Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gemspec
```

Comment:
Is it necessary to add a Gemfile ? does it help ? It can already imported in bundler application no ?

Reply:
Good question.

It's useful for development, but probably not necessary for normal usage.

If there is no `Gemfile`, you can't git clone the repo, run `bundle install` and start work on it. You'll have to manually install the dependencies.